### ISSUE TITLE
Creating Flows transformation

### ISSUE THREAD

**Issue by Gauhar** [Botmock/sdk#580]
@rahulchauhan049  - Can you please add some description to this

**Comment by rahulchauhan049** [comment]
This contains information on different flows including the `Default_Welcome_Flow` and `start` page/flow. This is an array of objects.

It contains information like `displayName`, `description`, `eventHandlers` and most important `transitionRoutes` which tells to which page does this connected to, including `conditions` and `intent` of the start page also. `CLI` later replaces the name of targetPage with the actual `id` of that page.

```jsx
[
	{
      "description":"A start flow created along with the agent",
      "displayName":"Default Start Flow",
      "etag":"",
      "eventHandlers":[
         {
            "event":"sys.no-match-default",
            "name":"989cd533-d3eb-4cf3-8e7d-b56caa759fe4",
            "triggerFulfillment":{
               "messages":[
                  {
                     "text":{
                        "text":[
                           "I didn't get that. Can you say it again?",
                           "I missed what you said. What was that?",
                           "Sorry, could you say that again?",
                           "Sorry, can you say that again?",
                           "Can you say that again?",
                           "Sorry, I didn't get that. Can you rephrase?",
                           "Sorry, what was that?",
                           "One more time?",
                           "What was that?",
                           "Say that one more time?",
                           "I didn't get that. Can you repeat?",
                           "I missed that, say that again?"
                        ]
                     }
                  }
               ]
            }
         },
         {
            "event":"sys.no-input-default",
            "name":"5b0bc721-f9d3-4165-a14e-106bec599445",
            "triggerFulfillment":{
               "messages":[
                  {
                     "text":{
                        "text":[
                           "I didn't get that. Can you say it again?",
                           "I missed what you said. What was that?",
                           "Sorry, could you say that again?",
                           "Sorry, can you say that again?",
                           "Can you say that again?",
                           "Sorry, I didn't get that. Can you rephrase?",
                           "Sorry, what was that?",
                           "One more time?",
                           "What was that?",
                           "Say that one more time?",
                           "I didn't get that. Can you repeat?",
                           "I missed that, say that again?"
                        ]
                     }
                  }
               ]
            }
         }
      ],
      "modelIdentifier":{
         "modelId":"b6e7ce5a-1dfe-46b1-965e-f46fbc8e01f9"
      },
      "name":"START_FLOW",
      "nluSettings":{
         "classificationThreshold":0.3,
         "customConfigKey":"d777bd08-55d0-4c3e-9a15-3fd3b1f65051",
         "modelType":"MODEL_TYPE_STANDARD"
      },
      "transitionRouteGroups":[
         
      ],
      "transitionRoutes":[
         {
            "condition":"",
            "intent":"DEFAULT_WELCOME_INTENT",
            "name":"29c92195-3e10-4aa5-8fb3-c568473bba7e",
            "targetPage":"getNamePage",
            "triggerFulfillment":{
               "conditionalCases":[
                  
               ],
               "messages":[
                  {
                     "text":{
                        "text":[
                           "Hi! How are you doing?",
                           "Hello! How can I help you?",
                           "Good day! What can I do for you today?",
                           "Greetings! How can I assist?"
                        ]
                     }
                  }
               ],
               "setParameterActions":[
                  
               ],
               "tag":"",
               "webhook":""
            }
         }
      ]
   }
]
```
